moral_kombat_backend/lead/home/api.py

```diff
@@ -0,0 +1,10 @@
+from home.models import Home
+from rest_framework import viewsets, permissions
+from .serializer import HomeSerializer
+
+class HomeViewSet(viewsets.ModelViewSet):
+    queryset = Home.objects.all()
+    permission_classes = [
+        permissions.AllowAny
+    ]
+    serializer_class = HomeSerializer
```

Comment:
I noticed this "No newline at end of file" this may be a Github-specific thing so I don't know if it may cause issues. 
From what I read it's good practice to add \n (newlines). In case of python this should be just inputting a new empty line
With JS this should be a semicolon(?) @mwnewm @sixbrigands maybe you know more about this. 

```suggestion
    serializer_class = HomeSerializer
  
```